packages/versioned-playground/src/VersionedSquigglePlayground.tsx

```diff
@@ -9,6 +9,11 @@ import {
   defaultSquiggleVersion,
 } from "./versions.js";
 
+export type ModelExport = {
+  variableName: string;
+  title?: string;
+};
```

Comment:
I was a bit unsure about this. Seemed like I should have it here, then have the Hub import it from here?